### PR TITLE
(plugin) notification::email - init new plugin

### PR DESCRIPTION
**Pull request by Tpo76**
Work is still in progress

New notification loking like this:

image

### CODE REVIEW

File: src/notification/email/mode/alert.pm [hunk 1]
```diff
@@ -0,0 +1,1133 @@
...
+use base qw(centreon::plugins::mode);
+
+use strict;
+use warnings;
+use LWP::UserAgent;
+use MIME::Base64;
+use Email::MIME;
+use Email::Sender::Simple qw(sendmail);
+use Email::Sender::Transport::SMTP;
+use JSON;
```
Comment (garnier-quentin) [inline src/notification/email/mode/alert.pm:32]: please use JSON::XS

File: src/notification/email/mode/alert.pm [hunk 2]
```diff
@@ -0,0 +1,1133 @@
...
+1;
+
+__END__
+
+=head1 MODE
+
+Send slack alerts.
+
+Example for a host:
+centreon_plugins.pl --plugin=notification::slack::plugin --mode=alert --slack-url='https://hooks.slack.com/services/T0A754E2V/B0E0CEL4B/81V8kCJusL7kafDSdsd' --slack-channel='#testchannel' --slack-username='bot' --slack-emoji=':ghost:' --host-name='srvi-clus-win' --host-state='DOWN' --host-output='test output' --priority='High' --zone='Production' --centreon-url='https://centreon.test.com/centreon/' --link-url='%{centreon_url}/main.php?p=20202&o=svc&host_search=%{host_name}'
```
Comment (garnier-quentin) [inline src/notification/email/mode/alert.pm:1032]: Please update the help. it's the slack notification help

**Review by garnier-quentin — COMMENTED** [review]
the file src/notification/email/mode/messages/services.pm doesn't seems used

**Comment by Tpo76** [comment]
Hello

Yes I'm still working on it, I still have a few ideas a few process can be the presentation can be enhanced.
Nevertheless this version is working fine.

I would also like to test the SMTP authentication process with Gmail and exchange365, but I'm missing the testing environment.

**Comment by garnier-quentin** [comment]
I have done some little updates. Could you test it ?

**Comment by Tpo76** [comment]
When service as no graph generateImage.php provide content like "OK u:0.01 s:0.01 r:0.00"
Triggering the '^OK' to select whether or not the graph must be included or not.

**Review by garnier-quentin — APPROVED** [review]

**Pull request merged**